lib/stackify/logger_client.rb

```diff
@@ -16,21 +25,37 @@ def log level, msg, call_trace
     end
 
     def log_exception level= :error, ex
-      if ex.is_a?(Stackify::StackifiedError)
-        Stackify::Utils.do_only_if_authorized_and_mode_is_on Stackify::MODES[:logging] do
+      case Stackify.configuration.transport
```

Comment:
Suggest changing the structure of the logger-client to have an interface an multiple implementations - one for api communication and one for unix domain socket.  This will avoid duplication of logic in each 'when' clause.